fs/glob.ts

```diff
@@ -43,3 +43,35 @@ export interface GlobOptions {
 export function glob(glob: string, options: GlobOptions = {}): RegExp {
   return globrex(glob, options).regex;
 }
+
+/** Test whether the given string is a glob */
+export function isGlob(str: string): boolean {
+  const chars: Record<string, string> = { "{": "}", "(": ")", "[": "]" };
+  const regex: RegExp = /\\(.)|(^!|\*|[\].+)]\?|\[[^\\\]]+\]|\{[^\\}]+\}|\(\?[:!=][^\\)]+\)|\([^|]+\|[^\\)]+\))/;
```

Comment:
The linter seems complaining on this line.
```
/home/vsts/work/1/s/fs/glob.ts
  50:9  error  Type RegExp trivially inferred from a RegExp literal, remove type annotation  @typescript-eslint/no-inferrable-types
```

Reply:
Is it okay now ?